stock_available_mrp/tests/test_potential_qty.py

```diff
@@ -12,6 +12,14 @@ class TestPotentialQty(TransactionCase):
     def setUp(self):
         super(TestPotentialQty, self).setUp()
 
+        self.product_model = self.env["product.product"]
+        self.bom_model = self.env["mrp.bom"]
+        self.bom_line_model = self.env["mrp.bom.line"]
+        self.stock_quant_model = self.env["stock.quant"]
+
+        self.setup_demo_data()
+
+    def setup_demo_data(self):
```

Comment:
@clonedagain Just to separate loading of models and datas.
(Some tests don't use demo datas. Although this data are still loaded, I can more easily see the part that interests me.)